packages/files-ui/src/Contexts/BillingContext.tsx

```diff
@@ -87,6 +89,12 @@ const BillingProvider = ({ children }: BillingContextProps) => {
     refreshInvoices()
   }, [refreshInvoices])
 
+  useEffect(() => {
+    if (!isLoggedIn) return
+
+    filesApiClient.getEligibility().then(res => setIsBillingEnabled(res.is_eligible))
```

Comment:
to be on the safe side
```suggestion
    filesApiClient.getEligibility()
      .then(res => setIsBillingEnabled(res.is_eligible))
      .catch(console.error)
```